frontend/src/components/ScheduledEventsTableCard/ScheduledEventsTableCard.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import axios from 'axios';
+import { Card } from 'react-bootstrap';
+import { Dimmer, Loader } from 'semantic-ui-react';
+import { ScheduledEventsTable } from '../ScheduledEventsTable/ScheduledEventsTable';
+import { api, localForageKeys } from '../../utils';
+import * as localForage from 'localforage';
+import { PaginationRow } from '../PaginationRow';
+
+export const ScheduledEventsTableCard = () => {
+  const [scheduledEvents, setScheduledEvents] = React.useState({ data: [] });
+  const [loading, setLoading] = React.useState(true);
+  const [count, setCount] = React.useState(0);
+  const [pageCount, setPageCount] = React.useState(0);
+  const [currentPage, setCurrentPage] = React.useState(1);
+  const [selectedPage, setSelectedPage] = React.useState(0);
+  const basePath = api.routes.VIEW_SCHEDULED_EVENTS;
+  const limit = 5;
+  React.useEffect(() => {
+    const fetchData = async () => {
+      const token = await localForage.getItem<string>(localForageKeys.API_KEY);
+      const headers = {
+        'Content-Type': 'application/json',
+        Authorization: `token ${token}`,
+      };
+      const result = await axios(`${basePath}`, { headers });
+      setScheduledEvents({ data: result.data });
+      setLoading(false);
+      setCount(result.data.length);
+      setPageCount(Math.ceil(result.data.length / 5));
+    };
+    fetchData();
+  }, []);
+
+  const handleScheduledEvents = () => {
```

Comment:
Could you give this a more descriptive name ... something like `getScheduledEventsByPage`, and move it out of the component ... takes in whatever you need as parameters.